_pages/en/research.md

```diff
@@ -5,4 +5,610 @@ lang: en
+
+[paragraph on main results]
+
+Results from the usability tests conducted in French identified similar usability issues to those found  in English testing. However, French speaking participants completed fewer tasks than english speaking participants, and reported a lower confidence level. This could be due to language and cultural differences. Another explanation for the difference could be that the French moderator was less experienced in conducting the usability tests. However, these are hypotheses that would require further study to validate. 
+
+Research in the beta phase was also conducted with frontline staff members from VAC area offices (Ottawa, Gatineau and Halifax) as well as the NCCN office in Halifax. As frontline staff work closely with Veterans to help them apply for benefits, the tool is useful as a resource for them to direct Veterans towards. Research insights confirm that staff are experts on the subject of VAC benefits and services and have lower needs to directly benefit from using the tool. However in sharing the tool with the Veterans they serve, staff members can be of greater help by answering their clients’ better informed, personalized questions. 
+
+
+## Methodology
+
+**Usability testing**
+
+A total of 4 moderated usability testing sessions were completed during the alpha phase, testing the tool with a total of 81 persons - i) Veterans, ii) still-serving military members, iii) family members of Veterans and still-serving military members, and iv) frontline staff, including case managers, veteran service agents and call centre agents.
+
+**Participant recruitment:**
+
+Beta test sessions 1, 2, and 3 were conducted inside VAC Area Offices located in Ottawa, Halifax and Gatineau respectively. Beta test session 4 was conducted in Petawawa, Ontario, out of recreational and community centres. 
+
+Veterans, still-serving military members and their family members were intercepted at these locations and requested to volunteer to participate. People were approached only once they had completed the tasks they come to the area office for. 
+
+If the moderator received a person's full consent to participate, they were guided into a comfortable, quiet room in the area office to begin their test session. 
+
+After completing the test, participants were thanked and directed to meet with a frontline staff member before they exited the premises. Consent forms and privacy documents were shared with them to take back. 
+
+**References and presentations**
```

Comment:
All these references need to be made public (and the docs that they reference as well), or need to be determined public or private and denoted in the doc as such.